paasta_tools/mesos_maintenance.py

```diff
@@ -76,6 +76,18 @@ def execute_master_api_request(method, endpoint, **kwargs):
     return execute_master_api_request
 
 
+def operator_api():
+    def execute_operator_api_request(**kwargs):
+        base_api_client = base_api()
+        if 'headers' in kwargs.keys():
```

Comment:
Is the `.keys()` necessary for some reason? Can't you just do `if 'headers' in kwargs`?